extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -87,7 +87,7 @@ require([
 				firstPostId: thread.firstPostId,
 				index: i,
 				link: '/d/p/' + thread.id,
-				shareUrl: 'http://' + window.location.hostname + '/d/p/' + thread.id,
+				shareUrl: window.location.protocol + '://' + window.location.hostname + '/d/p/' + thread.id,
```

Comment:
`window.location.protocol` includes the colon (i.e. `http:` and `https:`), so this results in `https:://`.